dotnet/samples/KernelSyntaxExamples/Example66_FunctionCallingStepwisePlanner.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Planners;
+using Plugins;
+using RepoUtils;
+
+// ReSharper disable once InconsistentNaming
+public static class Example66_FunctionCallingStepwisePlanner
+{
+    internal static string? ChatModelOverride = null; //"gpt-35-turbo-0613"
+
+    public static async Task RunAsync()
+    {
+        string[] questions = new string[]
+        {
+            "Write a limerick and send it via email to John"
+        };
+
+        var kernel = InitializeKernel();
+
+        var config = new FunctionCallingStepwisePlannerConfig
+        {
+
+        };
+        var planner = new FunctionCallingStepwisePlanner(kernel, config);
+
+        foreach (var question in questions)
+        {
+            FunctionCallingStepwisePlannerResult result = await planner.ExecuteAsync(question);
+            Console.WriteLine($"Q: {question}\nA: {result.FinalAnswer}");
+
+            // You can uncomment the line below to see the planner's process for completing the request.
+            // Console.WriteLine($"Chat history:\n{result.ChatHistory.AsJson()}");
+        }
+    }
+
+    /// <summary>
+    /// Initialize the kernel and load plugins.
+    /// </summary>
+    /// <returns>A kernel instance</returns>
+    private static IKernel InitializeKernel()
+    {
+        IKernel kernel = new KernelBuilder()
+            .WithLoggerFactory(ConsoleLogger.LoggerFactory)
+            .WithAzureOpenAIChatCompletionService(
+                ChatModelOverride ?? TestConfiguration.AzureOpenAI.ChatDeploymentName,
+                TestConfiguration.AzureOpenAI.Endpoint,
+                TestConfiguration.AzureOpenAI.ApiKey)
+            .Build();
+
+        kernel.ImportFunctions(new EmailPlugin(), "EmailPlugin");
```

Comment:
How come we don't need to import a function to write a limerick here (I am thinking something like the joke function)? If the step wise planner can perform certain tasks just by invoking the model, would be nice to document it here.

Reply:
Or would be nice to show a sample that requires multiple functions.

Reply:
I wanted to try something that used both the model's own capabilities and external functions. I'll add a comment to highlight that.

This sample does actually require/call multiple functions (get email address and send email), but I can explain that in a comment and also add a couple more examples.